data/campaigns/The_South_Guard/scenarios/05_Choice_In_The_Fog.cfg

```diff
@@ -273,6 +310,28 @@
             [objective]
                 description= _ "Death of Ethiliel"
                 condition=lose
+                [show_if]
+                    [variable]
+                        name=found_urza_afalas
+                        boolean_equals=no
+                    [/variable]
+                    [or]
+                        [variable]
+                            name=side_with_bandits
+                            equals=no
```

Comment:
I guess you meant to add the boolean prefix here too

Reply:
Fixup'd. I tested it and it works as intended even when side_with_bandits hasn't been set yet because empty values are treated as false.